plugins/modules/docker_image_pull.py

```diff
@@ -36,8 +34,8 @@
 options:
   name:
     description:
-      - "Image name. Name format must be one of V(name), V(repository/name), or V(registry_server:port/name).
-      - The name can optionally include the tag by appending V(:tag_name), or it can contain a digest by appending V(@hash:digest)."
+      - Image name. Name format must be one of V(name), V(repository/name), or V(registry_server:port/name). - The name can
+        optionally include the tag by appending V(:tag_name), or it can contain a digest by appending V(@hash:digest).
```

Comment:
```suggestion
      - Image name. Name format must be one of V(name), V(repository/name), or V(registry_server:port/name).
      - The name can optionally include the tag by appending V(:tag_name), or it can contain a digest by appending V(@hash:digest).
```